asset_dashboard/views.py

```diff
@@ -275,17 +275,23 @@ class AssetAddEditView(LoginRequiredMixin, TemplateView):
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
 
-        context['project'] = Project.objects.get(id=self.kwargs['pk'])
+        phase = Phase.objects.get(id=self.kwargs['pk'])
+
+        context.update({
+            'phase': phase,
+            'project': Project.objects.filter(phases=phase)[0],
+            'props': {
+                'phase_id': phase.id
```

Comment:
need this in `context['props']` so we can access the phase id in the react component.